private/model/api/operation.go

```diff
@@ -269,7 +269,7 @@ func (e *example) traverseAny(s *Shape, required, payload bool) string {
 		str = e.traverseStruct(s, required, payload)
 	case "list":
 		str = e.traverseList(s, required, payload)
-	case "map":
+	case "map", "jsonvalue":
 		str = e.traverseMap(s, required, payload)
```

Comment:
If the shape type is a `jsonvalue` does it still need to do the `traverseMap`? Might make sense to just create a serialized for JSONValue directly instead of reusing map, and not transversing.  In the case of JSONValue, while it is rendered as a map type the type is opaque, and there is never any transversing needed. Basically it is the same as a scalar as far as serializing here.